pdctl/command/store_command.go

```diff
@@ -30,14 +30,15 @@ var (
 // NewStoreCommand return a store subcommand of rootCmd
 func NewStoreCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   "store [delete|label|weight] <store_id>",
+		Use:   "store [delete|label|weight|set_namespace|rm_namespace] <store_id> [namespace]",
```

Comment:
I prefer using another subcommand here, not in store. maybe `store_ns set|rm`

Reply:
OK, I'll move it to another subcommand.